libraries/AP_Rally/AP_Rally.h

```diff
@@ -52,8 +54,15 @@ class AP_Rally {
 
     Location rally_location_to_location(const RallyLocation &ret) const;
 
+    // cmd_nav_rally parameters
+    bool cmd_go_to_rally_only = false;
+    bool cmd_go_to_rally_id = false;
+    uint8_t go_to_rally_id = 0;
```

Comment:
This is really bad, class variables shouldn't be public.

Reply:
Ok ! Will make them private and do method to access them